riscv_config/constants.py

```diff
@@ -12,10 +12,13 @@
         "Zfh",
         "Zfinx", "Zdinx", "Zhinx", "Zhinxmin",
         "Ztso",
-        "Zba", "Zbb", "Zbc", "Zbe", "Zbf", "Zbkb", "Zbkc", "Zbkx", "Zbm", "Zbp", "Zbr", "Zbs", "Zbt",
+        "Zba", "Zbb", "Zbc", "Zbe", "Zbf", "Zbkb", "Zbkc", "Zbkx", "Zbm", "Zbp", "Zbpbo", "Zbr", "Zbs", "Zbt",
         "Zk", "Zkn", "Zknd", "Zkne", "Zknh", "Zkr", "Zks", "Zksed", "Zksh", "Zkt",
+        "Zpn", "Zpsf",
         "Zmmul",
         "Svnapot"
 ]
+Zb_extensions = ["Zba", "Zbb", "Zbc", "Zbe", "Zbf", "Zbm", "Zbp", "Zbr", "Zbs", "Zbt"]
+Zp_extensions = ["Zbpbo", "Zpn", "Zpsf"]
```

Comment:
I'm a bit late to the party on this one, but is there any reason why these lists were added separately?
I mean it makes a lot of sense to split the Z list into the separate extensions they belong into, but then we should do that consistently and also append these lists to the main Z_extensions list before compiling isa_regex.

Reply:
It doesn't look like these two lists are really used anywhere else. I think they can be removed. As far as splitting it into separate extensions and merging them, I do not think that will work because the regex needs to be constructed canonically rather than by the groupings.

Reply:
This was added by a contributor to enable integration in their application. Its existence doesn't harm anything right now. But I see that this could snow-ball into way too many "harmless" snippets making its way here.

Reply:
I see. I was mainly scared of this snow-ball effect, and thought that if this catches on it might be just as good to actually define a separate list for each Z-extension group, which are all appended/concatenated into the final `Z_extensions` list that's used to build the regex.